about.html

```diff
@@ -0,0 +1,177 @@
+                    <i class='bx bx-menu'></i>
+                </div>
+
+                <div class="logo">
+                    <img src="assets/images/logo.png" alt="">
+                </div>
+
+            </div>
+
+            <ul>
+                <a  href="/">
+                    <li>Home</li>
+                </a>
+                <a class="active-link" href="about.html">
+                    <li>About</li>
+                </a>
+                <a href="#">
+                    <li>Join</li>
+                </a>
+                <a href="#">
+                    <li>Sponsor</li>
+                </a>
+                <a href="#">
+                    <li>News</li>
+                </a>
+            </ul>
+        </nav>
+
+        <div class="mobile-menu">
+            <div class="close-btn-container">
+                <i class='bx bx-x'></i>
+            </div>
+            <ul>
+                <a class="" href="/">
+                    <li>Home</li>
+                </a>
+                <a href="about.html">
+                    <li>About</li>
+                </a>
+                <a href="#">
+                    <li>Join</li>
+                </a>
+                <a href="#">
+                    <li>Sponsor</li>
+                </a>
+                <a href="#">
+                    <li>News</li>
+                </a>
+            </ul>
+        </div>
+    </header>
+
+    <div class="about-hero">
+        <h2>Who are we ?</h2>
+        <h1>EXPRESS YOURSELF THROUGH <br> YOUR LOOK</h1>
+
+        <div class="about-description">
+            <p>Discover the latest trends and embrace your unique style with our curated collection of fashion-forward
+                garments and accessories. We believe that fashion is an art form, allowing you to express your
+                individuality and creativity. Whether you're looking for chic everyday essentials, elegant eveningwear,
+                or statement pieces that make heads turn, we have something for every fashion enthusiast.
+            </p>
+        </div>
+
+        <div class="about-contact">
+            <p>Please contact us per Email for nay further questions about Yoka-Look</p>
+            <h3 class="about-contact-mail">YokaLook@.org</h3>
+        </div>
+    </div>
+
+    <div class="about-description">
+        <p>The Global Summit brings together the community of
+            experts, academics, and activists who comprise the Creative Commons affiliate network in a different country
+            every two years.
+            This year, we hope to expand our invitation list including organizations and individuals.</p>
+    </div>
+
+    <div class="about-contact">
+        <p>Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+        <h2>ccsummit2015@cckorea.org</h2>
+    </div>
+
+    <section class="sumit-program">
+        <h1 class="sumit-title">Creative Commons
+            Global Summit 2015 Logo
+        </h1>
```

Comment:
It is very important to have only one `h1` tag in the whole page in order to improve the SEO of the website, so kindly change other occurrence an `h1` tag except from the hero section.